packages/sui-widget-embedder/package.json

```diff
@@ -13,17 +13,16 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@s-ui/bundler": "7",
+    "@s-ui/bundler": "beta",
```

Comment:
Remind that draft for widget embedder https://github.com/SUI-Components/sui/pull/1308/files